import_mu.py

```diff
@@ -29,7 +29,7 @@
 
 from mu import MuEnum, Mu, MuColliderMesh, MuColliderSphere, MuColliderCapsule
 from mu import MuColliderBox, MuColliderWheel
-from shader import make_shader
+from shader import make_material
```

Comment:
if we want to report this in the trunk, it should probably be swtichable using a flag